### DISCUSSION TITLE
Jekyll tag pages only build locally but not with GitHub Pages

### DISCUSSION THREAD

**Discussion by myg-main** [community/community#21824] [Other Feature Feedback, Questions, & Ideas]
<p>Hi all,</p>
<p>I have a Jekyll blog. Each post has a tag that links to a “tag page” with a list of all the posts with the same tag.</p>
<p>It’s built locally but when I serve it up with GitHub pages, it links to the 404 page.</p>
<p>I thought it’s a timing issue(give GitHub pages some time and it will go away, which happens most of the time) but it doesn’t seem like it. </p>
<p>Can you give me some directions? Thanks in advance.</p>
<p>Here’s the repo: <a href="https://github.com/ming-yong/ming-yong.github.io" rel="nofollow noopener">https://github.com/ming-yong/ming-yong.github.io</a></p>
<p>Here’s the website: <a href="https://ming-yong.github.io/" rel="nofollow noopener">https://ming-yong.github.io/</a></p>
<p>Expected:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/2854ad3cc0e1fc0d082617ca85edab5a97095e72.png" data-download-href="/uploads/short-url/5KMw6x5vwow2rx3lHNIkxRFzmx4.png?dl=1" title="expect.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/43736500/181067031-e01ddefc-364a-4bda-a164-e5d15bbda0fd.png" alt="expect.PNG" data-base62-sha1="5KMw6x5vwow2rx3lHNIkxRFzmx4" width="690" height="264" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/2854ad3cc0e1fc0d082617ca85edab5a97095e72_2_690x264.png, https://aws1.discourse-cdn.com/github/optimized/1X/2854ad3cc0e1fc0d082617ca85edab5a97095e72_2_1035x396.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/2854ad3cc0e1fc0d082617ca85edab5a97095e72_2_1380x528.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/2854ad3cc0e1fc0d082617ca85edab5a97095e72_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">expect.PNG</span><span class="informations">1419×544 20.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Reality:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/948ac616c9287040b38409d551d9cd6f8a0dfbe5.jpeg" data-download-href="/uploads/short-url/lc3ZcnbfJDYz2ZDKpsnXKP9mPU9.jpg?dl=1" title="reality.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/43736500/181067033-0a6110e8-289b-453a-bd31-074922ba23f5.jpg" alt="reality.PNG" data-base62-sha1="lc3ZcnbfJDYz2ZDKpsnXKP9mPU9" width="690" height="298" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/948ac616c9287040b38409d551d9cd6f8a0dfbe5_2_690x298.jpg, https://aws1.discourse-cdn.com/github/optimized/1X/948ac616c9287040b38409d551d9cd6f8a0dfbe5_2_1035x447.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/948ac616c9287040b38409d551d9cd6f8a0dfbe5_2_1380x596.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/948ac616c9287040b38409d551d9cd6f8a0dfbe5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">reality.PNG</span><span class="informations">1489×644 141 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ChrisK91** [comment, marked as answer]
<p>The case sensitive link is working fine for me: <a href="https://ming-yong.github.io/tags/Year%20in%20Review/" rel="nofollow noopener">https://ming-yong.github.io/tags/Year%20in%20Review/</a></p>

**Comment by myg-main** [comment]
<p>I see! My files have names in uppercase. Thanks for spotting that, <a class="mention" href="/chrisk91">@chrisk91</a> !</p>
